packages/amplify-migration-tests/src/__tests__/migration_tests/api.connection.migration.test.ts

```diff
@@ -0,0 +1,85 @@
+import { initJSProjectWithProfile, deleteProject, amplifyPush, amplifyPushUpdate } from '../../../../amplify-e2e-tests/src/init';
+import { addApiWithSchema, updateApiSchema } from '../../../../amplify-e2e-tests/src/categories/api';
```

Comment:
Since this is used in both e2e and migration move this to e2e-core

Reply:
Included this in e2e core and changed the import